tests/ext/telemetry/metrics_logs_created.phpt

```diff
@@ -0,0 +1,199 @@
+--TEST--
+'logs_created' internal metric
+--SKIPIF--
+<?php
+if (getenv('PHP_PEAR_RUNTESTS') === '1') die("skip: pecl run-tests does not support {PWD}");
+if (getenv('USE_ZEND_ALLOC') === '0' && !getenv("SKIP_ASAN")) die('skip timing sensitive test - valgrind is too slow');
+?>
+--ENV--
+DD_TRACE_GENERATE_ROOT_SPAN=0
+DD_INSTRUMENTATION_TELEMETRY_ENABLED=1
+DD_TRACE_LOG_LEVEL=warn
+--INI--
+datadog.trace.agent_url="file://{PWD}/integration-telemetry.out"
```

Comment:
I missed that on the metrics.phpt, but it should be unique for all telemetry tests so that they can run in parallel. Can you please fix this?